lib/Service/SetupService.php

```diff
@@ -89,7 +89,7 @@ public function startSetup(IUser $user, string $gatewayName, string $identifier)
 		$verificationNumber = $this->random->generate(6, ISecureRandom::CHAR_DIGITS);
 		$gateway = $this->gatewayFactory->getGateway($gatewayName);
 		try {
-			$gateway->send($user, $identifier, "$verificationNumber is your Nextcloud verification code.");
+			$gateway->send($user, $identifier, "$verificationNumber is your verification code");
```

Comment:
:-1: we can talk about using the app name from theming, but I would keep it branded in general